docs/20_tutorial.md

```diff
@@ -1,64 +1,245 @@
 \pagebreak
 
-Run Book outline
-========
+# Run Book 
 
-***should the run book deal with the SOS establishing the risk
-limit, or any of their pre-audit responsibilities like the selection
-of which races to audit?
+## Introduction
 
-Brief introduction to RLAs in non-technical, human friendly language.
-At the county level, something along the lines of "This tool is going
-to help you cross reference votes recorded on paper ballots with how 
-the computers read those ballots, to determine whether the computers
-did their job accurately…” Try to put their minds at ease about the
-process. :)
+### What the RLA Tool Does
+
+The Risk-Limiting Audit (RLA) tool, developed by Free & Fair 
+for the Colorado Department of
+State for use in elections from November 2017 forward, supports a 
+Risk-Limiting Audit as required by Colorado statute and 
+as described in the Colorado Secretary of State's Rule 25 for
+elections. RLA-related terms and processes are defined and prescribed in Rule 25.
+*** citation for Rule 25.
+
+The RLA Tool enables the Secretary of State and each individual 
+County to carry out a Risk-Limiting Audit. The Secretary of State can
+enter the risk limit, the random seed and the contests that drive
+the audit process. The Secretary of State can also identify 
+contests that should go to a full hand count. Counties can upload
+ballot manifests and cast vote record (CVR) files. The
+RLA tool performs the necessary statistics (such as 
+choosing random samples of ballots for audit and determining 
+actual risk levels)
+County Audit Boards can 
+enter interpretations of ballots into the tool. And the Secretary of
+State can monitor the progress of the individual Counties.
+
+The RLA Tool also creates export files for use in the Audit
+Center, a website disseminating audit-related information
+to the public.
+ 
+### What the RLA Tool Does Not Do
+
+Although the RLA process requires Counties to 
+generate and preserve
+a Summary Results Report and a Results File Export, the RLA Tool 
+does not interact with these files in any way. Nor does the RLA Tool 
+aid in any way with the CVR Export Verification required by the 
+RLA process. 
+
+While the RLA Tool does provide for upload of the 
+hashed Ballot Manifest, the hashed CVR file and their hashes, 
+the RLA Tool does not provide a hashing utility.
+
+The RLA Tool does not provide any support
+for chain of custody logs. Nor does it provide any guidance for ballot 
+interpretation. 
+
+## For Secretary of State Users
+
+### Logging In
+
+Screenshot (SoS authentication) ---
+
+### Identifying County Users
+
+The Secretary of State will determine, for each individual
+County, which user will be able to log in to the County-facing
+part of the RLA Tool.
+
+Screenshot (SoS County User Identification)
```

Comment:
When the front end is done we'll review to make sure this kind of thing matches.